mkdocs.yml

```diff
@@ -93,9 +93,7 @@ nav:
       - 'Multicluster Gateways Walkthrough': multicluster-gateway-controller/docs/how-to/multicluster-gateways-walkthrough.md
       - 'Metrics Walkthrough': multicluster-gateway-controller/docs/how-to/metrics-walkthrough.md
     - 'Rate Limiting':
-      - kuadrant-operator/doc/user-guides/simple-rl-for-app-developers.md
-      - kuadrant-operator/doc/user-guides/gateway-rl-for-cluster-operators.md
```

Comment:
No RLP user guide for platform engineers anymore?

Reply:
good point, will keep kuadrant-operator/doc/user-guides/gateway-rl-for-cluster-operators.md